macos/FluentUI/Button/Button.swift

```diff
@@ -360,39 +374,47 @@ open class Button: NSButton {
 			backgroundColorRest = isWindowInactive ? ButtonColor.neutralBackground2 : accentColor
 			backgroundColorPressed = accentColor.withSystemEffect(.pressed)
 			backgroundColorDisabled = ButtonColor.brandBackgroundDisabled
-			borderColorRest = isWindowInactive ? ButtonColor.neutralStroke2 : .clear
-			borderColorPressed = .clear
-			borderColorDisabled = .clear
+			if !increaseContrastEnabled {
+				borderColorRest = isWindowInactive ? ButtonColor.neutralStroke2 : .clear
+				borderColorPressed = .clear
+				borderColorDisabled = .clear
+			}
```

Comment:
Nit (feel free to disagree), I'd rather move the check to per color, rather than one check where we set the colors again.

Reply:
Wouldn't it also be possible to just move the initial check for `increasedContrastEnabled` to the end, and not check at all in the switch?

Reply:
FWIW, my preference would be to continue the pattern already used and set the colors once inline. Makes future edits clear on where stuff happens vs. starting a new pattern.

Reply:
@huwilkes Yep I thought of that too - but felt a bit unnatural to set to one value and then overwrite it.

I just went with Saad's suggestion.